projects/packages/forms/src/dashboard/inbox/style.scss

```diff
@@ -298,12 +301,17 @@ $action-bar-height: 88px;
 	display: flex;
 	flex-direction: column;
 	height: 100%;
-	max-height: 100vh;
 	flex: 1 1 50%;
 
 	.dataviews-wrapper {
 		background: #fff;
-		border-radius: 8px;
+		height: auto;
+		overflow-y: visible;
+		overflow-x: auto;
+
+		@media (min-width: 782px) {
+			border-right: 1px solid var(--jp-gray-5);
+		}
```

Comment:
We needed a left border. Before, the vertical scrollbar acted as an effective right-hand border on the table (you can see in the screenshot). Once removed, the rows just ended into open space.